README.md

```diff
@@ -1,87 +1,25 @@
+1<img src="screenshot.png" width="800px">
```

Comment:
- [ ] Let's get rid of this HTML tag and use it the proper [markdown way](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#images). Also, I believe you have a typo in there :smile: